<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createAccountDialog</class>
 <widget class="QDialog" name="createAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>639</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>176</width>
        <height>183</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>SIGN UP!</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Email :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailLn"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="usernameLn"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>First Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="firstnameLn"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordLn"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="createAccountBtn">
         <property name="text">
          <string>create account</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Password</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>createAccountDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
